### ISSUE TITLE
Implement topic-specific GovDelivery signup form

### ISSUE THREAD

**Issue by jameshupp** [fecgov/fec-cms#1778]
The research around requirements for this issue happens in #1670. Definition of done is that we have a functional signup form on at least the Updates page.

- [ ] Incorporate the signup form into an accordion on the Updates page, aimed at the correct GovDelivery topic/category.
- [ ] Once the signup is in place, get the correct live topics from GovDelivery. (The ones in the code snippet they delivered are test topics.)
- [ ] Produce any documentation so this is a smoother, faster process on other pages that might need the same thing.

**Comment by jameshupp** [comment]
The requirements have shifted slightly since #1670 was closed, based on an earlier misunderstanding. Essentially, there are a series of user flow choices we can make. By far the simplest to manage is this one:

1. User opens accordion
1. User submits email address
1. User lands on a category page that corresponds to all Latest Updates topics, all topics in category are subscribed by default
1. User unselects any topics if desired

This addresses the following complications raised earlier, either in design syncs or in conversations with our GovDelivery rep:
- The behavior of the signup accordion—changing targets/language—is no longer incongruous with the location of the accordion being above the select boxes.
- Submitting _some_ topics but not _all_ topics within the category to the API and serving someone the proper landing page is only doable if we serve them a landing page that shows every single topic the FEC owns. With this solution, we can serve them only the Latest Updates category, so they can quickly and easily see which topics they're now getting.
- The user is subscribing to a category, rather than the individual topics. This makes changes and managing the topics easier:
  - A user who wishes to change subscriptions has a smaller universe to draw from.
  - The offices at the FEC who are responsible for the lists can quickly add new topics or adjust them within the GovDelivery universe; it requires no work by FEC's developers.

### tl;dr for what to build
- [ ] Simple email signup form accordion similar to [this one](https://user-images.githubusercontent.com/6314097/34270016-b355bfd6-e654-11e7-8995-33a0477f09ee.jpg)
- [ ] Actual interaction of that signup form based off [the code snippet we explored for the requirements](https://github.com/18F/fec-cms/files/1732654/USFEC_QSP1.txt)
- [ ] Copy for the accordion that makes clear the user is subscribing to all Latest Updates
- [ ] Flip of the switch by GovDelivery to make the correct category active

Once that list is complete, we can check off the first two boxes at the top of this issue. Then I'll put together some quick documentation about it and we'll be done.

**Comment by AmyKort** [comment]
Merged, so closing. Thanks!
